src/components/cards/LayoutCard.vue

```diff
@@ -11,9 +11,10 @@
   <v-card class="layout-card">
     <v-row
       align="center"
-      class="ml-5 mt-0"
+      class="ml-md-6 ml-1 my-1"
     >
-      <v-col>
+  
+      <v-col cols="auto">
         <h2 class="text-left layout-card-headline">{{ header }}</h2>
```

Comment:
```suggestion
        <h2 class="text-left headline-2">{{ header }}</h2>
```
Altlast, aber mit der Klasse ändert sich die Schriftgröße responsiv

Reply:
Bitte auch in main.scss die Zeilen 33-36 löschen.